webpack/base.js

```diff
@@ -33,9 +33,12 @@ const base = {
     */
     rules: [
       {
-        test: /\.(js|jsx)$/,
```

Comment:
Oh, and it's worth noting that this will cause issues with having regular JS files in the project. The original code handles both